client/src/components/DatasetInformation/DatasetError.vue

```diff
@@ -56,10 +56,11 @@
                     <h4 class="mb-3 h-md">Issue Report</h4>
                     <b-alert
                         v-for="(resultMessage, index) in resultMessages"
+                        v-html="resultMessage[0]"
```

Comment:
I think the resultMessage can potentially contain user-generated content, so that can open up XSS vulnerabilities and/or mess with the component layout.